R/DataDiscoveryRetrieval.R

```diff
@@ -316,23 +319,22 @@ TADABigdataRetrieval <- function(startDate = "null",
   if(!"null"%in%statecode&!"null"%in%huc){stop("Please provide either state code(s) OR huc(s) to proceed.")}
 
   if(!startDate=="null"){
-    startDate_Low = lubridate::ymd(startDate)
-    startYearLo = lubridate::year(startDate_Low)
+    startDate = lubridate::ymd(startDate)
+    startYearLo = lubridate::year(startDate)
   }else{ # else: pick a date before which any data are unlikely to be in WQP
-    startDate = "1800-01-01"
-    startDate_Low = lubridate::ymd(startDate)
-    startYearLo = lubridate::year(startDate_Low)
+    startDate = lubridate::ymd("1800-01-01")
+    startYearLo = lubridate::year(startDate)
   } 
   
 # Logic: if the input endDate is not null, convert to date and obtain year
   # for summary
   if(!endDate=="null"){
-    endDate_High = lubridate::ymd(endDate)
-    endYearHi = lubridate::year(endDate_High)
+    endDate = lubridate::ymd(endDate)
```

Comment:
Should we mention anything Laura D shared here about the nuisances between NWIS, WQP services, and dataRetrieval in the documentation here as well (e.g. endDate and startDate are equal to endDate_High, startDate_Low). This may help some users, but also ourselves (to refresh our memories on why we made these decisions). This could also just be added as comments in the function itself instead of the documentation.

Reply:
Added some details in the function description and in the vignette about date format. It is the one thing that isn't consistent between the WQP GUI and the package, but I think it's OK, given that the date format on the WQP GUI is a bit strange in the coding world. Let me know what you think.